vocab_files/attribute_concepts/habitat-description.ttl

```diff
@@ -7,13 +7,23 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/test/dawe-cv/f90dcce0-6d6c-4c75-8fd5-b09da3285b89>
     a skos:Concept ;
-    dcterms:source """McCallum K, Bignall J, Laws M, O’Neill S, Sparrow B. (unpublished draft) ‘Targeted Surveys module’ in ….. (eds) Ecological
-Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision
-making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
+    dcterms:source
+        "Laws M, McCallum K, Bignall J, O’Neill S, Sparrow B. (unpublished draft) Camera Traps Module in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ,
+        "McCallum K, Bignall J, Laws M, O’Neill S, Sparrow B. (unpublished draft) Targeted Surveys module in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ,
+        "McCallum K, Laws M, Bignall J, O’Neill S, Sparrow B. (unpublished draft) ‘Invertebrate module’ in ….. (eds) Ecological Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "A brief description of the habitat where the survey is being conducted." ;
+    skos:definition
+        "A brief description of each photo of the malaise trap habitat taken." ,
+        "A brief description of the habitat surrounding, where leaf-litter was sampled." ,
+        "A brief description of the habitat where invertebrates have been captured during active sampling." ,
+        "A brief description of the habitat where the camera traps have been deployed." ,
+        "A brief description of the habitat where the survey is being conducted." ,
+        "The areas of potential suitable habitat/s where the survey is being conducted. " ;
```

Comment:
Why is there more than one definition?

Reply:
This is an aggregation of all definitions, so that when Arun reviews definitions, he can decide which one is better or which word is good to preserve. I didn't delete any redundant definition (e.g. some definitions are only different from the module name) cuz not sure whether some words are useful to be part of the best definition.

Reply:
Ok cool, I'll remove myself from reviewer until this is finalised then.

Reply:
Why can't we merge it now, then Arun can decide and edit the best definition in main branch when doing the definition reviewing task?